gobgp/cmd/global.go

```diff
@@ -809,8 +811,21 @@ func ParsePath(rf bgp.RouteFamily, args []string) (*table.Path, error) {
 		if err != nil {
 			return nil, err
 		}
-		p := bgp.NewPathAttributeExtendedCommunities(extcomms)
+		normalextcomms := make([]bgp.ExtendedCommunityInterface, 0)
+		ipv6extcomms := make([]bgp.ExtendedCommunityInterface, 0)
+		for _, com := range extcomms {
+			bstring, _ := com.Serialize()
+			switch len(bstring) {
+			case 20:
```

Comment:
In that case, IPv6 Address Specific Extended Community cannot be distinguished from the existing community, Two-octet AS Specific Extended Community, because both have similar type value(0x00XX).
So, here I use length instead of type value.
Do you have any idea?